ng2-components/ng2-activiti-tasklist/src/components/activiti-people.component.spec.ts

```diff
@@ -232,43 +206,40 @@ describe('ActivitiPeople', () => {
             jasmine.Ajax.uninstall();
         });
 
-        it('should log error message when search fails', async(() => {
+        it('should log error message when search fails', () => {
             activitiPeopleComponent.peopleSearch$.subscribe(() => {
                 expect(logService.error).toHaveBeenCalledWith('Could not load users');
             });
             activitiPeopleComponent.searchUser('fake-search');
             jasmine.Ajax.requests.mostRecent().respondWith({
                 status: 403
             });
-        }));
+        });
 
-        it('should not remove user if remove involved user fail', async(() => {
-            activitiPeopleComponent.people.push(fakeUser);
-            fixture.detectChanges();
+        it('should not remove user if remove involved user fail', () => {
+            activitiPeopleComponent.people.push(...userArray);
             activitiPeopleComponent.removeInvolvedUser(fakeUser);
             jasmine.Ajax.requests.mostRecent().respondWith({
                 status: 403
             });
             fixture.whenStable()
                 .then(() => {
                     fixture.detectChanges();
-                    expect(element.querySelector('#user-fake-id')).not.toBeNull();
-                    expect(element.querySelector('#user-fake-id').textContent)
-                        .toBe('fake-name fake-last');
+                    expect(element.querySelector('.assignment-list-container')).not.toBeNull();
+                    expect(fixture.debugElement.queryAll(By.css('activiti-people-list alfresco-datatable tbody tr')).length).toBe(2);
                 });
-        }));
+        });
 
-        it('should not involve user if involve user fail', async(() => {
-            activitiPeopleComponent.involveUser(fakeUserToInvolve);
+        it('should not involve user if involve user fail', () => {
```

Comment:
i don't think the test is testing correctly the behaviour without the async clause here.